File: app/src/main/java/im/point/dotty/network/LoginReply.java
package im.point.dotty.network;

import com.google.gson.annotations.SerializedName;

public final class LoginReply extends Envelope {
    String token;
    @SerializedName("csrf_token") String csrfToken;

    public LoginReply(String token, String csrfToken) {
        this.token = token;
        this.csrfToken = csrfToken;
    }

    public String getToken() {
        return token;
    }

    public String getCsrfToken() {
        return csrfToken;
    }
}

File: app/src/main/java/im/point/dotty/network/LoginReply.kt
/*
 * Copyright (c) 2019-2021 Pavel A. Sokolov
 */
package im.point.dotty.network

import com.google.gson.annotations.SerializedName

data class LoginReply(var token: String? = null,
                      @SerializedName("csrf_token")
                      var csrfToken: String? = null) : Envelope()